presidio-anonymizer/presidio_anonymizer/deanonymize_engine.py

```diff
@@ -34,3 +29,19 @@ def get_deanonymizers(self) -> List[str]:
         """Return a list of supported deanonymizers."""
         names = [p for p in self.operators_factory.get_deanonymizers().keys()]
         return names
+
+    def add_deanonymizer(self, deanonymizer_cls: Type[Operator]) -> None:
+        """
+        Add a new deanonymizer to the engine.
+
+        anonymizer_cls: The deanonymizer class to add to the engine.
+        """
```

Comment:
Do we want to add log here? As the add and remove de-anonymizers are not that common?

Reply:
Good idea. I'll add

Reply:
Fixed